packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -522,6 +521,8 @@ export function PrescribeWorkflow(props: PrescribeProps) {
               }
             });
 
+            combineOrSubmit();
```

Comment:
was calling the wrong function; `combineOrSubmit` delegates to appropriate downstream functions like `combine`; `submitForm` is the nuclear option and i was taking it.